lib/Driver/ToolChains.h

```diff
@@ -45,10 +45,14 @@ class LLVM_LIBRARY_VISIBILITY GenericUnix : public ToolChain {
 
   virtual std::string getDefaultLinker() const;
 
+  virtual std::string getTarget() const;
```

Comment:
I probably should have required this on the last commit, but all of these need doc comments describing the default behavior.